contact.html

```diff
@@ -12,10 +14,125 @@
 
 <body>
 
-	<h1>Contact</h1>
+	<nav>
+		<div class="hamburger">
+			<h2>S&J</h2>
+			<img class="nav-expand" src="img/nav-hamburger.png" alt="open/close">
+		</div>
+		<div class="nav-links">
+			<div class="nav-header-overlay">
+				<h2>S&J</h2>
+				<img class="navclose" src="img/nav-hamburger-close.png" alt="close">
+			</div>
+			<div class="nav-overlay-links">
+				<a href="index.html">Home</a>
+				<a href="services.html">Services</a>
+				<a href="#">Contact</a>
+			</div>
+		</div>
+	</nav>
+
+	<div class="contact-header">
+		<h1>Contact Us</h1>
+	</div>
+
+	<section class="contact">
+		<div class="left-contact">
+			<h2>Get In Touch</h2>
+
+			<form action="">
+				<h4>Email address</h4>
+				<input type="email" name="email" placeholder="Enter email"><br>
```

Comment:
for accessibility, you want the to use the `<label>` tag, and use the `for` attribute properly